src/browser/modules/D3Visualization/components/DetailsPane.test.tsx

```diff
@@ -123,4 +128,36 @@ describe('<DetailsPane />', () => {
       screen.getByRole('button', { name: 'Show 2 more' })
     ).toBeInTheDocument()
   })
+
+  test('should handle show more on long property value', async () => {
```

Comment:
It's great that you've added this test!

Do you think it's worth adding tests for values both larger and smaller than the `WIDE_VIEW_THRESHOLD` ?

Maybe one test would do `nodeInspectorWidth={WIDE_VIEW_THRESHOLD - 1}` and the other with `WIDE_VIEW_THRESHOLD + 1` or something?

Reply:
Sounds like a good idea. I'll add a test for that.